<?xml version="1.0"?>
<launch>
    <arg name="camera_topic" default="" />
    <arg name="queued_image_topic" default="queued_image_topic" />
    <arg name="action_detected_topic" default="action_detected" />
    <arg name="action_validation_topic" default="action_validation" />
    <node pkg="action_recognition" type="image_queuing.py" name="image_queuing_node" output="screen">
        <param name="camera_topic" type="string" value="$(arg camera_topic)" />
        <param name="queued_image_topic" type="string" value="$(arg queued_image_topic)" />
        <param name="action_detected_topic" type="string" value="$(arg action_detected_topic)" />
        <param name="action_validation_topic" type="string" value="$(arg action_validation_topic)" />
    </node>
</launch>
